CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+* Port to Cesium 1.7.1 and Ol 3.3.0
+
+# Breaking changes
+  * Require the `olcs.OLCesiium.setEnabled` parameter to be defined.
```

Comment:
Should be "The `enable` argument to `olcs.OLCesium#setEnabled()` is no longer optional."